src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,79 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type PropsPagination = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}: PropsPagination) => {
+  const items = getNumbers(1, total);
+  const totalPages = Math.ceil(items.length / perPage);
+  const totalPagesArr = getNumbers(1, totalPages);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn(
+          'page-item',
+          { disabled: currentPage === 1 },
+          { active: currentPage === 1 },
+        )}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => {
+            if (currentPage !== 1) {
+              onPageChange(currentPage - 1);
+            }
+          }}
+        >
+          «
+        </a>
```

Comment:
Using `href="#prev"` and `href="#next"` may lead to unwanted behavior in the URL. Consider using `button` elements instead of `a` elements for pagination control to avoid this issue.

Reply:
If mentor's written HTML elements and structure like this I'm not gonna touch it, my task is different